src/main/java/com/google/sps/servlets/Shared.java

```diff
@@ -0,0 +1,58 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query.Filter;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+
+public class Shared {
+
+  public final String CORRELATOR = "correlator";
+  public final String FIND_NEAREST_LOCATION = "clicksFindNearestLocation";
+  public final String GRANTS_LOCATION = "grantsLocation";
+  public final String INTERACTS_WITH_MAP = "interactsWithMap";
+  public final String SKIP_TO_CONTENT = "clicksSkipToContent";
+  public final String RETURN_TO_AD = "clicksReturnToAd";
+  public final String INTERACTION_TABLE = "Interactions";
+
+ /**
+  * Given an HTTP request object, this method will get the correlator parameter
+  * and return it. Returns an empty string if correlator is not given.
+  */
+  public static String getCorrelator(HttpServletRequest request) {
+    String correlator = request.getParameter(CORRELATOR);
+
+    if (correlator == null) {
+      return "";
+    }
+
+    return correlator;
+  }
+ 
+ /**
+  * Given the correlator and the property to update, this will update the database to reflect that the user
+  * clicked on the corresponding metric.
+  *
+  * correlator - A correlator of type String for the current user.
+  * propertyToUpdate - A property of type String that is to be updated
+  */
+  public static void updateDatabase(String correlator, String propertyToUpdate) {
```

Comment:
A more common name might be DatabaseUtilities, DBUtil, Utilities, DataHandler.

A more common way to name methods is to try and be more specific about what it actually does for example setToTrue.

That way your function call would be
DataBaseUtil.setToTrue(correlator, Constant.FIND_NEAREST_LOCATION);

Pretty clear exactly what that line does, whereas updateDatabase is ambiguous (update to what)

Reply:
Okay great. I have named one class DBUtilities and the other Property to make it clear